raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -228,7 +230,7 @@ scenario:
           name: "Assert after 100 payments from 3 to 0"
           tasks:
             - wait: 100
-            - assert_sum: {from: 0, balance_sum: 1_995_000_000_000_000_000}
+            - assert_sum: {from: 0, balance_sum: 1_105_000_000_000_000_000}
```

Comment:
What the reason for this change? If I remember correctly this scenarios has successfully run several times in the past, with the previous value?

Reply:
Please read the linked issue and/or the commit messages. The scenario has not ran succesfully for weeks now.

TL;DR: One of the channels is closed and settled and as such its balance is not taken into account.

Reply:
Yeah, just saw the issue comment afterwards. (For some reason I expected the reasoning of the PR to be explained in the PR itself and not the issue)

> The scenario has not ran succesfully for weeks now.
I am aware, but was just wondering why it ever passed in the first place then.

Reply:
> I am aware, but was just wondering why it ever passed in the first place then.

Because the settlement timeout has gotten reduced to make scenario runs faster, in combination with something (no idea what yet) in Raiden making things go slower resulting in slow scenario runs.

Reply:
A PR description would have easier to reason about the change:

> Finally to make the scenario pass we wait for 40 blocks to make sure that the settlement timeout has passed and the channel0-4 is settled and as such taken out of the balance calculation. And we adjust the asserted balance to 1_105_000_000_000_000_000.

Reply:
Read the issues or the commit messages which you copied pasted.